webextension/privileged/ui/cloudstorage.js

```diff
@@ -63,6 +59,15 @@ this.cloudstorage = class extends ExtensionAPI {
           return path;
         },
 
+        cleanUpPrefs() {
```

Comment:
is this used anywhere?

Reply:
As of now it's not and commented in background.js. https://github.com/mozilla/shield-cloudstorage/pull/38/commits/e5f56cd380b632fc9ff9469e23febceede38e89a#diff-7f87af4f00f7747a008141453c7eb047R64

I can remove it before shipping